README.md

```diff
@@ -0,0 +1,66 @@
+# `react-native-shake-to-report`
+
+<img src="src/assets/screenshot.png" width="852" />
+
+Using this library, you can add to your app the shake event listener,
+which will trigger a custom list of options, each one with it's own logic.
+When the 'Report a problem' option is selected an email with the device's
+information will be sent or a callback function will pe triggered, depending
+on the method provided.
+
+## Installation
+
+```sh
+npm install react-native-shake-to-report
+```
+
+or
+
+```sh
+yarn add react-native-shake-to-report
+```
+On iOS, install cocoapods:
+
+```bash
+cd ios
+pod install
+```
+
+## Usage
+
+### Example
+
+```javascript
+import Shkr from 'react-native-shake-to-report';
+```
+
+```javascript
+const extraItems = [{
+  icon: extraItemIcon,
+  title: "Extra Item Title",
+  action: handleExtraItem,
+}];
+
+const reportIssue = (deviceInfo) => {
+  // your code here
+}
+
+  <Shkr
+    items={extraItems}
+    email='example@example.com'
+    reportIssue={(deviceInfo) => reportIssue(deviceInfo)}
+  />
+```
+
+### Props
+
+| Prop name      | Type     | Description                                                                                                                       |
+|----------------|----------|-----------------------------------------------------------------------------------------------------------------------------------|
+| items          | array    | An array of objects, each one having the 'icon', 'title' and 'action' keys. Their values will be added to the list of options.    |
```

Comment:
"Their values will be added to the list of options." -> "Each object represents a menu option."?